pkg/validations/runner.go

```diff
@@ -27,6 +30,7 @@ func (r *Runner) Run() error {
 			failed = true
 		}
 	}
+	r.validations = make([]Validation, 0)
```

Comment:
what is this for?

Reply:
This is so I can register() and run() validations multiple times using the same runner. Just trying to clear out the r.validations array after a run.